File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.java

class Solution {
    public int solution(String my_string, String is_suffix) {
        String[] answer = new String[my_string.length()];
        for(int i = 0; i < my_string.length(); i++){
            answer[i] = my_string.substring(i, my_string.length());
        }
        
        for(int i = 0; i < answer.length; i++){
            if(is_suffix.equals(answer[i])) return 1;
        }
        return 0;
    }
}

File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.js
function solution(my_string, is_suffix) {
    let answer = [];
    for(let i = 0; i < my_string.length; i++){
        answer.push(my_string.slice(i, my_string.length))
    }
    for(const str of answer){
        if(str == is_suffix) return 1
    }
    return 0;
}